mne/epochs.py

```diff
@@ -1810,6 +1810,56 @@ def save(self, fname, split_size='2GB', fmt='single', overwrite=False,
             this_epochs.event_id = self.event_id
             _save_split(this_epochs, fname, part_idx, n_parts, fmt)
 
+    def export(self, fname, fmt="auto"):
+        """Export Epochs to EEGLAB's .set format.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension
+
+        Notes
+        -----
+        Channel locations are expanded to the full EEGLAB format
+        For more details see eeglabio.utils.cart_to_eeglab
+        """
+        if fmt == "auto":
+            fmt = op.splitext(fname)[1]
+            if fmt:
+                fmt = fmt[1:]
+            else:
+                raise ValueError("Couldn't infer format from filename "
+                                 "(no extension found)")
+
+        if fmt == "set":
+            from eeglabio.epochs import export_set
```

Comment:
one more comment: see how we do it with `_check_pandas_installed`, we should do something similar here.